docs/quickstart.md

```diff
@@ -3,61 +3,138 @@ hide:
   - navigation
 ---
 
-This tutorial will introduce the main NeMoS functionalities. This is intended for users that are 
-already familiar with the GLM framework but want to learn how to interact with the NeMoS API. 
-If you have used [scikit-learn](https://scikit-learn.org/stable/) before, we are compatible with the [estimator API](https://scikit-learn.org/stable/modules/generated/sklearn.base.BaseEstimator.html), so the quickstart should look 
-familiar.
+At his core, NeMoS consists of two primary modules: the `basis` and the `glm` module.
 
-In the following sessions, you will learn:
+The `basis` module focuses on designing model features (inputs) for the GLM. It includes a suite of composable feature 
+constructors that accept time-series data as inputs. These inputs can be any observed variables, such as presented 
+stimuli, head direction, position, or spike counts. 
 
-1. [How to define and fit a GLM model.](#basic-model-fitting)
-2. [What are the GLM input arguments.](#model-arguments)
-3. [How to use NeMoS with `pynapple` for pre-processing.](#pre-processing-with-pynapple)
-4. [How to use NeMoS with `scikit-learn` for pipelines and cross-validation.](#compatibility-with-scikit-learn)
+The basis objects can perform two types of transformations on the inputs:
 
-Each of these sections can be run independently of the others.
+1. **Non-linear Mapping:** This process transforms the input data through a non-linear function, 
+   allowing it to capture complex, non-linear relationships between inputs and neuronal firing rates. 
+   Importantly, this transformation preserves the properties that makes GLM easy to fit and guarantee a 
+   single optimal solution (e.g. convexity).
 
-### Basic Model Fitting
+2. **Convolution:** This applies a convolution of the input data with a bank of filters, designed to 
+   capture linear temporal effects. This transformation is particularly useful when analyzing data with 
+   inherent time dependencies or when the temporal dynamics of the input are significant.
 
-Defining and fitting a NeMoS GLM model is straightforward:
+Both transformations produce a vector of features `X` that changes over time, with a shape 
+of `(n_time_points, n_features)`.
+
+On the other hand, the `glm` module maps the feature to spike counts. It is used to learn the GLM weights, 
+evaluating the model performance, and explore its behavior on new input.
+
+### Model Fitting
+
+Here's a brief demonstration of how the `basis` and `glm` modules work together within NeMoS.
+
+#### Poisson GLM for features analysis
+
+<img src="../assets/glm_features_scheme.svg" width="100%">
+
+In this example, we'll construct a time-series of features using the basis objects, applying a non-linear mapping
+(default behavior):
```

Comment:
```suggestion
In this example, we'll construct a time-series of features using the basis objects.
```

If you accept my suggestion about removing the bit about non-linear mapping vs. convolutoin